src/migrations/20200814190000_meta.js

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
```

Comment:
GitHub isn't displaying this one as a rename, but it is a rename. The "deleted" file is shown above.